src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+abstract class Machine {
+    public abstract void doWork();
+
+    public abstract void stopWork();
+

```

Comment:
For the future, remove the last empty line in all classes/interfaces/methods where it is present